angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html

```diff
@@ -57,24 +58,24 @@
 	</div>
 	<div class="row no-margin"  ng-if='!loading && !error'>
 		<div class="col-xs-12 col-sm-4 col-lg-3 desc-item">
-			<h3 class="center">Description</h3>
-			<p class="app-description">{{::portlet.description}}</p>
+			<h3 tabindex="0" aria-label="Description" class="center">Description</h3>
```

Comment:
That was what I thought before I tested it.
all the content could not be read out in the description. However,
sometimes, different screen reader have different ability. Like for JAWS,
we could use up and down arrow to let it read the content, but as a people
who could not know where is the header or content (which level it is), how
could he know where is the content. I am confusing about it, too.

On Mon, Aug 10, 2015 at 5:44 PM, Andrew Petro <notifications@github.com>
wrote:

> In
> angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html
> <https://github.com/UW-Madison-DoIT/angularjs-portal/pull/297#discussion_r36693835>
> :
>
> > @@ -57,24 +58,24 @@
> >  	</div>
> >  	<div class="row no-margin"  ng-if='!loading && !error'>
> >  		<div class="col-xs-12 col-sm-4 col-lg-3 desc-item">
> > -			<h3 class="center">Description</h3>
> > -			<p class="app-description">{{::portlet.description}}</p>
> > +			<h3 tabindex="0" aria-label="Description" class="center">Description</h3>
>
> This is fine if it's necessary, but it makes me worried about how poorly I
> understand accessibility, since i would have hoped the Description in the
> content of the <h3 element would have been good enough to get a screen
> reader to understand that "Description" is the level-three heading here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UW-Madison-DoIT/angularjs-portal/pull/297/files#r36693835>
> .
>